component/operator.libsonnet

```
@@ -0,0 +1,70 @@
+// main template for kube_prometheus
```

If we have file comments, they should be correct ;)

```suggestion
// template for prometheus-operator
```